README.md

```diff
@@ -92,19 +98,14 @@ For check results in GitHub Markdown syntax you can use --ghm:
 
     fontbakery check-googlefonts --verbose *.ttf --ghm
 
-### FontBakery web Dashboard
-
-There is a web dashboard that is used for monitoring the check-results of the full Google Fonts collection (or possibly other collections of font families). This tool was initialy developed in this repository, but later it was split out into its own git repo, now available at: https://github.com/googlefonts/fontbakery-dashboard
-
 ### Automated testing of all Google Fonts
 
 If you need to generate a list of all issues in a font family collection, such as the Google Fonts collection, you have to have a full checkout of it, and then you can run:
 
-```
-sh bin/fontbakery-check-collection.sh path-to-collection-directory
-```
+    sh bin/fontbakery-check-collection.sh path-to-collection-directory
 
-This will create a folder called check_results and it will run fontbakery on every family from the collection, thus generating individual per-font-file reports both in json and in github markdown format. The reports are saved in subdirectories that have the names of the families.
+This will create a folder called `check_results/`, then run Font Bakery `check-googlefonts` on every family from the collection. 
+The output is individual per-font-file reports, both in json and in github markdown format, each saved in subdirectories that are the names of the families.
```

Comment:
We don't have markdown yet. I also believe that the json output that we can generate is a) not made this way, b) not accessible at all via CLI at the moment